integration/makefile.mk

```diff
@@ -0,0 +1,32 @@
+GOFAIL_BINARY = $(shell pwd)/gofail
+
+.PHONY: run-all-integration-tests
+run-all-integration-tests: build-gofail run-integration-test-sleep clean-gofail
+
+.PHONY: clean-all-integration-tests
+clean-all-integration-tests: clean-integration-test-sleep
+
+# run integration test - sleep
+.PHONY: run-integration-test-sleep
+run-integration-test-sleep: build-integration-test-sleep execute-integration-test-sleep clean-integration-test-sleep
+
+.PHONY: build-integration-test-sleep
+build-integration-test-sleep:
+	cd ./integration/sleep/failpoints && ${GOFAIL_BINARY} enable && cd .. && go build -o integration_test_sleep . && cd failpoints && ${GOFAIL_BINARY} disable
```

Comment:
Can you follow the same pattern to simplify the makefile?
https://github.com/etcd-io/etcd/blob/260dd77ed41ba31c65d1d5608156d4548fce346e/tests/robustness/makefile.mk#L51-L52

Reply:
It's OK to continue to enhance the makefile in followup PRs. You call

Reply:
I will perfect it before we merge and tag for 0.2.0!

Reply:
Gentle ping @ahrtr :)